contrib/platform/src/com/sun/jna/platform/win32/Kernel32Util.java

```diff
@@ -81,6 +81,90 @@ public static void freeGlobalMemory(Pointer ptr) {
     }
 
     /**
+     * Closes all referenced handles. If an exception is thrown for
+     * a specific handle, then it is accumulated until all
+     * handles have been closed. If more than one exception occurs,
+     * then it is added as a suppressed exception to the first one.
+     * Once closed all handles, the accumulated exception (if any) is thrown
+     *
+     * @param refs The references to close
+     * @see #closeHandleRef(HANDLEByReference)
+     */
+    public static void closeHandleRefs(HANDLEByReference... refs) {
```

Comment:
I don’t think there’s a problem understanding it when simply named “closeHandles"; the goal is to close handles, regardless of the form used to specify the handles.  

I generally don’t like codifying parameter types into function names.

> On Mar 9, 2016, at 9:48 AM, Lyor Goldstein <notifications@github.com> wrote:
> 
> In contrib/platform/src/com/sun/jna/platform/win32/Kernel32Util.java:
> 
> > @@ -81,6 +81,90 @@ public static void freeGlobalMemory(Pointer ptr) {
> >      }
> >  
> >      /**
> > +     * Closes all referenced handles. If an exception is thrown for
> > +     * a specific handle, then it is accumulated until all
> > +     * handles have been closed. If more than one exception occurs,
> > +     * then it is added as a suppressed exception to the first one.
> > +     * Once closed all handles, the accumulated exception (if any) is thrown
> > +     *
> > +     * @param refs The references to close
> > +     * @see #closeHandleRef(HANDLEByReference)
> > +     */
> > +    public static void closeHandleRefs(HANDLEByReference... refs) {
> 
> I thought about that, but decided it would be easier to understand the code - if you feel strongly about it I have no major objection to renaming it to closeHandles
> 
> —
> Reply to this email directly or view it on GitHub.
>